specs/protocol/jovian/exec-engine.md

```diff
@@ -128,9 +139,12 @@ It takes on a default value as described in the section on [L1 Attributes](./l1-
 
 ### Rationale
 
-While the current L1 fee mechanism charges for DA usage based on an estimate of the DA footprint of a transaction, it
-does not influence future base fee calculations. As a result, excessive DA usage is not efficiently reflected in the fee
-market, leading to suboptimal resource prices.
-
-By changing the meaning of the `gasUsed` field in times of high DA demand, the fee market can properly adjust without reverting
-to priority fee auctions (an inferior experience for users).
+While the current L1 fee mechanism charges for DA usage based on an estimate of the DA footprint of a transaction, no
+protocol mechanism currently reflects the limited _DA throughput_. E.g. on Ethereum with Pectra enabled, the available
+blob throughput is `~96 kB/s` (with a target of `~64 kB/s`), but the calldata floor gas price of `40` for calldata-heavy
+transactions allows for more incompressible transaction data throughput than even that of most single OP Stack chains.
```

Comment:
This isn't quite scanning for me. Is the point that the calldata pricing on L2 is such that, with incompressible data, a single OP Stack chain can produce more DA pressure per unit time than Ethereum mainnet can accomodate?

Reply:
OK. I think what is throwing me off is that we are talking about a throughput being _higher_ (more) than an OP Stack chain. But I was expecting you to compare to an L1 chain. Perhaps you can try to rephrase it?

Reply:
Now I see that the sentence didn't make sense at all 😅 rewritten this section, PTAL

Reply:
Nice, works for me now!